tests/playwright/specs/wikipediaHomePage.js

```diff
@@ -1,14 +1,21 @@
+const { saveVideo } = require( 'playwright-video' );
+const path = require( 'path' ),
+	LOGPATH = process.env.LOG_DIR || path.resolve( __dirname, '..', 'log' );
+
 describe( 'Page', () => {
-	beforeAll( async () => {
+	it( 'should display sub title correctly', async () => {
+		let capture;
 		await page.goto( 'https://www.wikipedia.org/' );
-	} );
 
-	afterAll( async () => {
-		await browser.close();
-	} );
+		capture = await saveVideo( page, `${LOGPATH}/recording.mp4` );
```

Comment:
Move this line to line 7, so video recording starts before the page is loaded.

Reply:
This has been changed.